modules/Microsoft.Network/bastionHosts/.test/addpip.parameters.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
@elbatane is there any indication to why this used to work? For example, did the provider change?

Reply:
@MrMCake - I was having this conversation with @eriqua - We had some issues with bastion and sub-resources that was deployed "behind the scenes" my understanding is the provider had made some changes not covered in the API Versions to simplify deployment and fix the cross subscription routing issues faced when deploying manually.

I could be wrong but its a hunch